src/net-questdb-client/Senders/TcpSender.cs

```diff
@@ -63,6 +63,7 @@ private void Build()
        _buffer = new Buffer(Options.init_buf_size, Options.max_name_len, Options.max_buf_size);
 
        var socket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, System.Net.Sockets.ProtocolType.Tcp);
+       //socket.SendTimeout = 60 * 1000;
```

Comment:
This looks like a leftover, is it?

Reply:
Leftover 😁
Before I found out that it did not work for Async.
Actually I don't know if it would work for the sync call. Or if the CancelationTokenSource.CancelAfter works for the sync call.